impexp-client/src/main/java/org/citydb/gui/modules/preferences/view/PreferencesPanel.java

```diff
@@ -242,11 +232,11 @@ public void doTranslation() {
 		restoreButton.setText(Language.I18N.getString("common.button.restore"));
 		standardButton.setText(Language.I18N.getString("common.button.standard"));
 		applyButton.setText(Language.I18N.getString("common.button.apply"));
-		noticeLabel.setText(Language.I18N.getString("common.pref.menu.expand.label"));
+		hintLabel.setText(Language.I18N.getString("common.pref.menu.expand.label"));
 
 		resetPreferencesMenu();
-		menuTree.repaint();
-		prefLabel.setText(((PreferencesTreeNode)menuTree.getLastSelectedPathComponent()).toString());	
+		settingsNameLabel.setText(menuTree.getLastSelectedPathComponent().toString());
+		splitPane.resetToPreferredSizes();
```

Comment:
`splitPane.repaint();` would be better. The `resetToPreferredSizes` sometimes crushes the left tree panel when switching preferences content panels.

Reply:
What does "crushing" mean? I haven't experienced this so far. `resetToPreferredSizes()` is required for resizing, which `repaint()` doesn't do. So maybe adding `repaint()` helps to avoid the "crushing", but we need to keep `resetToPreferredSizes()`.

Reply:
For example: see the screenshot below (Note that there is a horizontal scroll bar at the bottom of the preferences content panel)
![image](https://user-images.githubusercontent.com/5942971/100994795-3f93a680-3557-11eb-849b-058f6d8f3251.png)
Now, if you click another tree node e.g. "gml:id Behandlung", and then click the "Fortschreibung" node again, the GUI will be shown like this: The tree panel has been _crushed_.
![image](https://user-images.githubusercontent.com/5942971/100995425-0ad41f00-3558-11eb-914b-05b2010aeb3b.png)

Reply:
Does not seem to happen all the time though, only sometimes. Would be nice if you can find a solution that keeps `resetToPreferredSizes()` :-)

Reply:
addressed in 1a3c7a284213268ff6895a9bcbf7d430b5ce159c